.env.example

```diff
@@ -0,0 +1,2 @@
+AZURE_AI_MISTRAL_LARGE_ENDPOINT=MISTRAL_LARGE_ENDPOINT
```

Comment:
Is it correct to include `AZURE` in the environment variable name? It's very possible that users will be running this code outside of Azure.

I'm wondering if we namespace env variables to make them specific to our app `INFER_GPT....` to avoid any confusion? Happy to over-ruled if we disagree.